pkg/controllers/support.go

```diff
@@ -212,3 +214,15 @@ func (l logSink) WithName(name string) logr.LogSink {
 func FilteredLogger(logger logr.Logger) logr.Logger {
 	return logger.WithSink(logSink{logger.GetSink()})
 }
+
+var (
+	hashConstant = "codeflare-operator"
+)
+
+func ControllerSpecificHash(s string, l int) string {
```

Comment:
I'd be tempted to not have the hash length as an argument. WDYT?

Reply:
In that case should we just leave it defined by a variable at `8`?

Reply:
Yes, like adding a `hashLength` variable along side the `hashConstant`.